src/core/agent/services/credentialService.ts

```diff
@@ -192,109 +193,14 @@ class CredentialService extends AgentService {
     });
   }
 
-  private async updateAcdcMetadataRecordCompleted(
-    id: string,
-    cred: any
-  ): Promise<CredentialShortDetails> {
-    const metadata = await this.getCredentialMetadataByCredentialRecordId(id);
-    if (!metadata) {
-      throw new Error(CredentialService.CREDENTIAL_MISSING_METADATA_ERROR_MSG);
-    }
-
-    metadata.status = CredentialMetadataRecordStatus.CONFIRMED;
-    metadata.credentialType = cred.schema?.title;
-    await this.updateCredentialMetadata(metadata.id, metadata);
-    return this.getCredentialShortDetails(metadata);
-  }
-
-  private async getKeriNotificationRecordById(
-    id: string
-  ): Promise<KeriNotification> {
-    const result = await this.basicStorage.findById(id);
-    if (!result) {
-      throw new Error(`${CredentialService.KERI_NOTIFICATION_NOT_FOUND} ${id}`);
-    }
-    return {
-      id: result.id,
-      createdAt: result.createdAt,
-      a: result.content,
-    };
-  }
-
   async deleteKeriNotificationRecordById(id: string): Promise<void> {
```

Comment:
Can this be moved to the `signifyNotificationService`? It doesn't make too much sense here